src/main/java/thePackmaster/cards/pickthemallpack/BigBoots.java

```diff
@@ -0,0 +1,60 @@
+package thePackmaster.cards.pickthemallpack;
+
+import com.evacipated.cardcrawl.mod.stslib.cards.interfaces.OnObtainCard;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.cards.blue.Stack;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardAndObtainEffect;
+import thePackmaster.SpireAnniversary5Mod;
+
+public class BigBoots extends AbstractPickThemAllCard implements OnObtainCard {
+    public static final String ID = SpireAnniversary5Mod.makeID("BigBoots");
+    private static final int COST = 1;
+    private static final int DRAW = 2;
+    private static final int UPGRADE_DRAW = 1;
+    private static final int THRESHOLD = 15;
+
+    public BigBoots() {
+        super(ID, COST, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        this.magicNumber = this.baseMagicNumber = DRAW;
+        this.secondMagic = this.baseSecondMagic = THRESHOLD;
+        this.cardsToPreview = new Stack();
```

Comment:
Might be too much effort, but I don#t see any logic to hid this card preview when in combat. Not sure if it needs to exist once the card is in the deck at all.

Reply:
That logic exists, it's in `AbstractPickThemAllCard.resetDescriptionForCombat`, because all card previews in this pack are for pickup effects so they can all be removed for combat.